docs/kqueen.rst

```diff
@@ -153,6 +153,110 @@ in the configuration file, set the environment variable matching the KQUEEN_<con
       - Addresses allowed to access metrics endpoint without token
 
 
+External services deployment
+-------
```

Comment:
An intro needed. For example:

Before you provision a Kubernetes cluster, you may need to deploy and configure the following external services:

* Members management (describe the details)
* Metrics collecting (describe the details)

**To set up members management:**

1.
2.
3.

**To set up metrics collecting:**

1.
2.
3.